lib/ui/profitloss/RecordAmountHeader.dart

```diff
@@ -0,0 +1,117 @@
+import 'package:farmsmart_flutter/ui/common/RegExInputFormatter.dart';
+import 'package:farmsmart_flutter/ui/profitloss/RecordAmount.dart';
+import 'package:flutter/material.dart';
+import 'package:flutter/services.dart';
+
+class RecordAmountHeaderViewModel {
+  String amount;
+  final Function(String) listener;
```

Comment:
naming, what does it do? at a guess..   onAmountChanged?

Reply:
Okey, changed